src/main/java/Schmidt.java

```diff
@@ -0,0 +1,219 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
```

Comment:
Dude, honestly your code is super good and follows the code rule. I have not a lot of comments but good job!!!

Reply:
Thanks 😁